app-admin/etcd-wrapper/etcd-wrapper-0.0.1.ebuild

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2015 CoreOS, Inc.. All rights reserved.
+# Distributed under the terms of the GNU General Public License v2
+# $Header:$
+#
+
+EAPI=6
+
+DESCRIPTION="etcd Wrapper"
+HOMEPAGE="https://github.com/coreos/etcd"
+KEYWORDS="amd64"
```

Comment:
I only left it out since it depends on rkt. I guess it doesn't hurt to include it - it just won't run properly yet.